packages/fluent/scss/daterangepicker/index.scss

```diff
@@ -0,0 +1,27 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "daterangepicker",
+    dependencies: (
+        "input",
+        "floating-label",
+        "popup",
+        "icon"
```

Comment:
add dependency to button here. There is a case where a swap button in rendered between the inputs